kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -40,21 +37,18 @@ import (
 	"knative.dev/eventing-contrib/kafka/source/pkg/reconciler/resources"
 
 	eventingv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
-
-	"knative.dev/pkg/logging"
+	eventingclientset "knative.dev/eventing/pkg/client/clientset/versioned"
 
 	// NewController stuff
-	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/kubernetes"
 	appsv1listers "k8s.io/client-go/listers/apps/v1"
-	"k8s.io/client-go/tools/cache"
-	clientset "knative.dev/eventing-contrib/kafka/source/pkg/client/clientset/versioned"
+	versioned "knative.dev/eventing-contrib/kafka/source/pkg/client/clientset/versioned"
+	reconcilerkafkasource "knative.dev/eventing-contrib/kafka/source/pkg/client/injection/reconciler/sources/v1alpha1/kafkasource"
 	listers "knative.dev/eventing-contrib/kafka/source/pkg/client/listers/sources/v1alpha1"
 	eventinglisters "knative.dev/eventing/pkg/client/listers/eventing/v1alpha1"
-	"knative.dev/eventing/pkg/reconciler"
-	"knative.dev/pkg/resolver"
-
-	"knative.dev/pkg/controller"
 	pkgLogging "knative.dev/pkg/logging"
```

Comment:
I would switch this to just be logging or no alias

Reply:
Done in 62ab377